Cue/app/tabs/library/LibraryHome.js

```diff
@@ -125,6 +126,44 @@ class LibraryHome extends React.Component {
   }
 
   _onPressAddDeck = () => {
+    Alert.alert(
+      Platform.OS === 'android' ? 'Add deck' : 'Add Deck',
+      'You can create a new private deck or add a shared deck by entering a share code.',
+      [
+        {text: 'Cancel', style: 'cancel'},
+        {text: 'Look up share code', onPress: this._onPressLookUpDeckByShareCode},
+        {text: 'Create new deck', onPress: this._onPressCreateDeck},
```

Comment:
I am low-key high-key very upset at React Native right now. Hooray for working around quirks in React Native! (Also: Title Case on the button labels.)

<img width="755" alt="screen shot 2017-03-25 at 3 49 44 am" src="https://cloud.githubusercontent.com/assets/13400887/24320575/53df85f8-110e-11e7-8575-6f1ed8838971.png">